app/api/api/views.py

```diff
@@ -197,6 +197,19 @@ class UserFilterViewSet(viewsets.ReadOnlyModelViewSet):
     filterset_fields = {"id": ["in", "exact"], "is_active": ["exact"]}
 
 
+class CurrentUserViewSet(viewsets.ReadOnlyModelViewSet):
```

Comment:
I think this is done better with an APIView, rather than a Model View.

Something like:

```
class UserDetailView(APIView):
    permission_classes = [IsAuthenticated]

    def get(self, request, *args, **kwargs):
        serializer = UserSerializer(request.user)
        return Response(serializer.data)
```

I'm keen we don't squeeze in obvious API views into model views

Reply:
(this would also mean it's added in `urls.py` not `router.py` btw)